### ISSUE TITLE
[feature request] Some activations modules missing inplace argument

### ISSUE THREAD

**Issue by vadimkantorov** [pytorch/pytorch#28962]
E.g. nn.Tanh and nn.Sigmoid are missing inplace argument, though torch.Tensor.sigmoid_ exists (although torch.nn.functional.sigmoid does not support inplace/out argument either)

https://pytorch.org/docs/master/tensors.html?highlight=sigmoid_#torch.Tensor.sigmoid_

consistency wrt out arguments is nice to easier support inplace mode like in https://github.com/pytorch/pytorch/issues/26644#issuecomment-534762427
